circle.yml

```diff
@@ -436,24 +436,24 @@ workflows:
         jobs:
```

Comment:
Was this meant to change? I just followed the instructions in https://github.com/cypress-io/cypress-docker-images/blob/master/CONTRIBUTING.md

Reply:
Yeah this changes depending on what the last generated image is.